extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java

```diff
@@ -281,9 +282,12 @@ long importPhotoBatch(
         }
       }
     } catch (IOException e) {
-      if (e.getMessage() != null
-          && e.getMessage().contains("The remaining storage in the user's account is not enough")) {
+      if (StringUtils.contains(e.getMessage(), "The remaining storage in the user's account is not enough")) {
         throw new DestinationMemoryFullException("Google destination storage full", e);
+      } else if (StringUtils.contains(e.getMessage(), "The provided ID does not match any albums")) {
+        // which means the album was likely deleted by the user
+        // we skip this batch and log some data to understand it better
```

Comment:
nit (another non-blocking comment): the inline comment you're leaving here makes it seem like a mystery we'll follow back up on, but from our chat today I got the impression you're pretty confident you know why this is happening a lot and there's nothing we plan to do about It? Maybe "we skip this batch and log some data so we can monitor that the rate of occurrence doesn't start increasing" or something like that?